webserver.Tests/Project/Controllers/REControllerTest.cs

```diff
@@ -0,0 +1,301 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Mvc;
+using webserver.Controllers;
+using webserver.Models;
+using webserver.Data;
+using webserver.Models.DTOs;
+
+namespace webserver.Tests.Project.Controllers {
+    public class REControllerTest {
```

Comment:
Please, use full names.
When the app gets bigger, this will make difference
```suggestion
    public class RealEstateControllerTest {
```